scripts/stats/hurricane/exevs_hurricane_global_ens_spread_stats.sh

```diff
@@ -112,7 +111,7 @@ export enddate="$YY02$MM02$DD02$HH02"
 echo "$startdate, $enddate"
 
 #--- run for TC_pairs
-cp ${PARMevs}/metplus_config/${STEP}/${COMPONENT}/TCPairs_template.conf .
+cp ${PARMevs}/metplus_config/stats/hurricane/TCPairs_spread_template.conf .
```

Comment:
@OliviaOstwald-NOAA Would it be possible for this line to use variables again, as it used to? Like ${STEP}/${COMPONENT} instead of stats/hurricane? 

CC @malloryprow

Reply:
Yep! I will look for other lines where I may have missed any hardcoded variables too, but feel free to point any out that you come across before I do.